### PR TITLE
Cleanup for elixirconf

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
-`nerves_key_quickstart_fw` - Nerves firmware to run on common raspberry pi targets.
+`nerves_key_quickstart_fw` - Nerves firmware to run on official Nerves targets.
 
 Building and running this locally:
 
 ```bash
 cd nerves_key_quickstart_phx
 mix deps.get
-cd assets && npm install && cd ..
+cd assets && npm install && npm run deply && cd ..
```
Comment (jjcarstens) [inline README.md]: Should this be `deploy`?

Comment (fhunleth) [reply README.md]: Yes. Thanks.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,45 @@
...
+The NervesKey Quickstart firmware is an example firmware to help provision
+NervesKey devices. For large scale deployments you'll want to integrate
+NervesKey provisioning into your manufacturing process, but for one-off
+provisioning, this firmware can save some time.
+
+To try it out, download a prebuilt firmware image from the GitHub releases page
+and copy it to a MicroSD card for your device. Raspberry Pis and Beaglebones are
+supported.
+
+Next, power up your device and connected to it via wired Ethernet or by a USB
```
Comment (jjcarstens) [inline README.md]: `and connect`

Comment (fhunleth) [reply README.md]: Fixed.

**Pull request merged**